chainstate/src/chainstate_interface.rs

```diff
@@ -26,8 +26,22 @@ pub trait ChainstateInterface: Send {
         height: &BlockHeight,
     ) -> Result<Option<Id<Block>>, ChainstateError>;
     fn get_block(&self, block_id: Id<Block>) -> Result<Option<Block>, ChainstateError>;
+
+    /// Returns an exponential sequence of block headers.
```

Comment:
I would call it "a list of block hashes whose heights distances increase exponentially starting from the current tip"